core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -152,7 +148,7 @@ class ClientQuotaManager(private val config: ClientQuotaManagerConfig,
    * Returns the consumer quota for the specified clientId
    * @return
    */
-  private[server] def quota(clientId: String): Quota = overriddenQuota.getOrElse(clientId, defaultQuota)
+  def quota(clientId: String): Quota = overriddenQuota.getOrDefault(clientId, defaultQuota)
```

Comment:
getOrDefault() seems to be in java 8 only? Also, the comment in line 148 seems inaccurate. This is not just a consumer quota.